controller/ingress_entry.go

```diff
@@ -50,6 +50,9 @@ func (e IngressEntry) validate() error {
 	if e.ServiceAddress == "" {
 		return errors.New("missing service address")
 	}
+	if e.ServiceAddress == "None" {
+		return errors.New("Service address is set to 'None' ignoring entry")
```

Comment:
The error is handled upstream so I don't think we need the 'ignoring entry' part. Also, could we have the log all lower case too for consistency when printing (as this will be appended to an error) so we will just have `service address is set to 'None'`. After that I think this is good to go.

Reply:
Also, when releasing can you update this PR to merge into the `add-merlin-hostnames` branch so we can then have one PR which is merging into master with one changelog and one release.

Reply:
cool :+1: